server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/RegexMappingRoutes.java

```diff
@@ -48,35 +73,36 @@ public String getBasePath() {
 
     @Override
     public void define(Service service) {
-        service.post(getBasePath(), this::addRegexMapping);
+        service.post(ADD_ADDRESS_MAPPING_PATH, this::addRegexMapping);
     }
 
     @POST
-    @Path(BASE_PATH)
+    @Path(ADD_ADDRESS_MAPPING_PATH)
     @ApiOperation(value = "adding address-regex mappings to RecipientRewriteTable")
     @ApiResponses(value = {
-        @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "No body created"),
-        @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "Invalid body")
+        @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "Mapping successfully added"),
+        @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "Invalid parameter")
```

Comment:
Idem

Reply:
I updated the message to be more precise and clearer.